### PR TITLE
New post: Invitation to test : IR Hosting

### PR DESCRIPTION
**Pull request by mmo**

### CODE REVIEW

File: content/post/invitation-to-test.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Invitation to test : IR hosting service"
+date: 2020-09-28T17:00:00+02:00
+draft: false
+tags: ["iraas", "repository", "shared portal", "dedicated portal"]
+---
+
+One of the goals of the [SONAR project](https://sonar.ch/) is to set up a hosting service for institutional repositories (IRaaS - Institutional Repository as a Service). [Two variants](https://sonar.ch/iraas/) of this service are proposed: a shared portal and dedicated portals. In anticipation of its first production launch, this service is currently providing a public test instance at the address [sonar.test.rero.ch](https://sonar.test.rero.ch).
```
Comment (PascalRepond) [inline content/post/invitation-to-test.md]: In anticipation of its first production launch, this service is currently providing
to 
In anticipation of its first production launch, the service is currently providing

File: content/post/invitation-to-test.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+<!--more-->
+
+In order to respond to the interest that it has been attracting in our community, the SONAR team is inviting all interested institutions to test this service. Just send a short message to [info.sonar@rero.ch](mailto:info.sonar@rero.ch) with the **name of the institution** and the **name and email address of the person in charge** (who will be assigned the [Admin role](https://sonar.test.rero.ch/help/user_roles/)).
+
+
+<img class="image fit" src="/images/invitation-to-test.png" alt="SONAR public test portal" title="SONAR public test portal" >
+
+
+The [help section](https://sonar.test.rero.ch/help/) provides explanations about the different user roles and the document deposit and validation process. Some features are still under development, and will progressively become visible. The development backlog is available (live) on [GitHub](https://github.com/orgs/rero/projects/4).
```
Comment (PascalRepond) [inline content/post/invitation-to-test.md]: The [help section](https://sonar.test.rero.ch/help/) provides explanations about the different user roles and the document deposit and validation process.
to
The [help section](https://sonar.test.rero.ch/help/) provides explanations about the different user roles, as well as document deposit and validation processes.

**Review by PascalRepond — CHANGES_REQUESTED** [review]

File: content/post/invitation-to-test.md (hunk 2 above)
Comment (mmo) [reply content/post/invitation-to-test.md]: It's a single process involving deposit and validation.

File: content/post/invitation-to-test.md (hunk 1 above)
Comment (mmo) [reply content/post/invitation-to-test.md]: OK.

File: content/post/invitation-to-test.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+date: 2020-09-28T17:00:00+02:00
+draft: false
+tags: ["iraas", "repository", "shared portal", "dedicated portal"]
+---
+
+One of the goals of the [SONAR project](https://sonar.ch/) is to set up a hosting service for institutional repositories (IRaaS - Institutional Repository as a Service). [Two variants](https://sonar.ch/iraas/) of this service are proposed: a shared portal and dedicated portals. In anticipation of its first production launch, this service is currently providing a public test instance at the address [sonar.test.rero.ch](https://sonar.test.rero.ch).
+
+<!--more-->
+
+In order to respond to the interest that it has been attracting in our community, the SONAR team is inviting all interested institutions to test this service. Just send a short message to [info.sonar@rero.ch](mailto:info.sonar@rero.ch) with the **name of the institution** and the **name and email address of the person in charge** (who will be assigned the [Admin role](https://sonar.test.rero.ch/help/user_roles/)).
```
Comment (MarionRERO) [inline content/post/invitation-to-test.md]: Ici je mettrais plutôt: "the interest the project is raising in our community" ou bien "So as the community can get acquainted with the future service, the SONAR team....to test it."

**Pull request merged**